cafebabel/tests/test_drafts.py

```diff
@@ -32,6 +32,7 @@ def test_create_draft_should_generate_article(client, user, editor):
         'author': user.id
     }, follow_redirects=True)
     assert response.status_code == 200
+    assert '<span>en</span>' in response
```

Comment:
Still passing the language within the test? What about hitting `/es/article/draft/new/` to be sure it's not the default that is rendered?